components/List.vue

```diff
@@ -6,97 +6,100 @@
       @input="updatePage"
       circle
     ></v-pagination>
-    <v-row no-gutters>
+    <v-row>
       <v-col
         v-for="(article, index) in historyList"
         :key="index"
         sm="2"
         md="6"
         lg="4"
       >
-        <v-sheet class="pa-2 ma-2">
-          <v-card width="100%" class="list-img">
-            <v-img :src="article.urlToImage"></v-img>
-            <v-card-title
-              ><a :href="article.url">{{ article.title }}</a></v-card-title
-            >
-            <v-card-text>{{ article.author }}</v-card-text>
-            <v-card-text
-              ><h4>Published date :</h4>
-              20th January 2023</v-card-text
-            >
-            <button @click="$router.push(`/details/${article.title}`)">
-              view Details
-            </button>
-          </v-card>
-        </v-sheet>
+        <!-- <v-sheet class=""> -->
+        <v-card height="100%">
+          <v-img height="40%" :src="article.urlToImage"></v-img>
+          <v-card-title
+            ><a :href="article.url">{{ article.title }}</a></v-card-title
+          >
+          <v-card-text>{{ article.author }}</v-card-text>
+          <v-card-text
+            ><h4>Published date :</h4>
+            20th January 2023</v-card-text

```

Comment:
please render the published date that is there in the response for each article and format it as `20th January 2023`

Reply:
Resolved